docs/pages/blog/migration-update.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Why you should migrate to Material UI v5 today
+description: We have completely revamped our Migration guide to reduce friction when upgrading to v5. Get started now!
+date: 2022-06-15T00:00:00.000Z
+authors: ['samuelsycamore']
+card: true
+tags: ['News', 'MUI Core']
+---
+
+Are you still using Material UI v4 in 2022?
+
+What are you thinking?! 😛
+
+We released v5 in late 2021, and since then, we've seen a steady migration of users over to the new packages.
```

Comment:
?

```suggestion
We [released v5 in late 2021](/blog/mui-core-v5/), and since then, we've seen a steady migration of users over to the new packages.
```